docs/conf.py

```diff
@@ -66,7 +66,8 @@
 
 html_show_sourcelink = True
 html_sidebars = {
-    "**": ["logo-text.html", "globaltoc.html", "localtoc.html", "searchbox.html"]
+    "**":
```

Comment:
That was done by the formatter. I have now added the `.style.yapf` file from fpga-tool-perf so it will match the general Symbiflow formatting. After making this change, the formatter fails, I will look more into the failure and try to fix it.